src/components/modals/modalSendMail/index.tsx

```diff
@@ -0,0 +1,167 @@
+import { sendForm } from '@emailjs/browser';
+import {
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+// eslint-disable-next-line import/no-named-as-default
+import ReCAPTCHA from 'react-google-recaptcha';
+import { toast } from 'react-hot-toast';
+
+import { Icon } from '@components/icon';
+import { cn } from '@utils/helpers';
+
+import {
+  type IModal,
+  Modal,
+} from '../modal';
+
+interface IModalGithubLogin extends Pick<IModal, 'onClose'> {
+  title: string;
+}
+
+const publicKey = import.meta.env.VITE_EMAIL_PUBLIC_KEY;
```

Comment:
One general question, are there any secrets there and if so shouldn't we consider making this from the backend. Also .env.example should be updated